webrtc-stats.html

```diff
@@ -1167,7 +1167,14 @@
               <dd>
                 <p>
                   Only valid for audio, and the value is between 0..1 (linear), where 1.0
-                  represents <code>0 dBov</code>. Calculated as defined in [[!RFC6464]].
+                  represents <code>0 dBov</code>, 0 represents silence, and 0.5 represents
+                  approximately 6 dBSPL change in the sound pressure level from 0 dBov.
+                </p>
+                <p>
+                  The "audio level" value defined in [[!RFC6464]] (defined as 0..127, where 0
```

Comment:
Used "is obtained" rather than "can be obtained". Made 6465 normative and 6464 informative.